templates/energyplus/templates/ref-sys.pxt

```diff
@@ -121,3 +126,122 @@ Refrigeration:Condenser:AirCooled,
     22.2;                    !- Maximum Value of x
 <% end %>
 
+#Afshin Faramarzi and Behzad Rizi, 11/11/2024 - added EnergyManagementSystem to the model to control saturated condensing temperature 
+!-   ===========  ALL OBJECTS IN CLASS: ENERGYMANAGEMENTSYSTEM:ACTUATOR ===========
+
+<% if (ref_system_name == "MT_System") %>
```

Comment:
Instead of the if-statement here, can we substitute each instance of "MT" or "MT_System" with the parameter ref_system_name? The root file will run this template once each for LT_System and MT_System. For example,

    EnergyManagementSystem:Actuator,
        CondensingTempOverride<%= ref_system_name %>,!- Name
        <%= ref_system_name %>,               !- Actuated Component Unique Name
        Refrigeration:System,    !- Actuated Component Type
        Minimum Condensing Temperature;  !- Actuated Component Control Type

Reply:
I have addressed your comment and removed the if-statement for "ref_system_name". Now, we just have one if-statement for "condenser_type".